src/main/java/app/linguistai/bmvp/exception/GlobalExceptionHandler.java

```diff
@@ -41,7 +41,7 @@ public ResponseEntity<Object> handleSomethingWentWrongException(SomethingWentWro
     @ResponseStatus(HttpStatus.NOT_FOUND)
     @ResponseBody
     public ResponseEntity<Object> handleCustomException(CustomException ex) {
-        return Response.create(ex.getMessage(), HttpStatus.BAD_REQUEST);
+        return Response.create(ex.getMessage(), HttpStatus.NOT_FOUND);
```

Comment:
I think the default HTTP status being "Bad Request" is a better design choice. I see that a few lines above, there is a line `@ResponseStatus(HttpStatus.NOT_FOUND)` which is conflicting with the original function body. It might me better to change this instead, what do you think?

Reply:
Bad request is handled in the function below, called handleValidationException. My understanding was that handleCustomException was specifically for Not Found error - what do you think @elif-krvan

Reply:
I believe you may be correct, let's merge if @elif-krvan agrees

Reply:
I think is a copy paste error. This method should catch all custom errors, including  the not found error. As much as I can remember all custom exceptions have an http status. So instead of HttpStatus.something we can do ex.getStatus(). In that case, we also need to remove @ResponseStatus above.

Reply:
Changed it - is this correct? @elif-krvan @can-aslan